README.md

```diff
@@ -28,6 +29,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | [Critical Software](https://www.criticalsoftware.com) [:rocket:](https://www.criticalsoftware.com/careers) | Mission and business-critical applications. | `Coimbra` `Lisboa`<br>`Porto` |
 | [Equal Experts](https://www.equalexperts.com) [:rocket:](https://www.equalexperts.com/contact-us/lisbon/) | Making software. Better. | `Lisboa` `Remote` |
 | [Imaginary Cloud](https://www.imaginarycloud.com) [:rocket:](https://www.imaginarycloud.com/careers) | Web, mobile, development and design services. | `Lisboa` |
+| [JTA - The Data Scientists](https://www.thedatascientists.com/) [:rocket:](https://www.thedatascientists.com/careers) | Machine Learning , Reporting and custom vizualizations services | `Porto` |
```

Comment:
Some issues here, can you please:
1. Shorten the company name so that layout does not break?
2. Remove extra ` ` space before the `,` comma in the description;
3. End the description sentence with a `.` period;
4. Fix the careers link which currently returns a `HTTP 404` error.

Thanks.